src/pm/hydra/tools/topo/topo.c

```diff
@@ -46,9 +46,6 @@ HYD_status HYDT_topo_init(char *user_topolib, char *user_binding, char *user_map
     else if (MPL_env2str("HYDRA_MEMBIND", &membind) == 0)
         membind = NULL;
```

Comment:
There are more usages of getting parameters via environment here, they should suffer the same issues as `HYDRA_TOPO_DEBUG`, right?

Reply:
I see that the other 4 parameters -- `topolib`, `binding`, `mapping`, `membind` -- are both parsed from command line and environment, the latter probably doesn't work similarly. I think I understand your comment of `all proxy parameters` better now.  I guess for consistency, adding `topo-debug" to the command line list is plausible. But we need also also parse the other 4 environment variables in `mpiexec` too, right? 

Honestly, if we can, I would like to get rid of the environment options (eliminating `HYDRA_MEMBIND` ... all 5 of them) and just advise user to use command line options.

Reply:
Yes, these are most likely all broken. I'd be happy to remove them, as all our documentation uses the command line options. The only user-reported issue we've received is for `HYDRA_TOPO_DEBUG`.